src/exchanges/bitfinex-client.js

```diff
@@ -188,7 +206,9 @@ class BitfinexClient extends BasicClient {
   }
 
   _onTradeMessage(msg, market) {
-    let [, , , id, unix, price, amount] = msg;
+    // example msg: [ 359491, 'tu', [ 560287312, 1609712228656, 0.005, 33432 ], 6 ]
+    let [id, unix, amount, price] = msg[2];
```

Comment:
We have a regression with v2 @ line 220.  The timestamp in v1 was in seconds, the timestamp in v2 is milliseconds.  Line 220 needs to have the `* 1000` removed.